docs/src/modules/java/pages/kickstart.adoc

```diff
@@ -138,7 +138,7 @@ The Akka Serverless Maven plugin creates a unit test stub for each class. Use th
 .src/test/java/com/example/domain/CounterTest.java
 [source,java]
 ----
-include::example$archetype/first-service/src/test/java/com/example/domain/CounterTest.java[]
+//FIXME include::example$archetype/first-service/src/test/java/com/example/domain/CounterTest.java[]
```

Comment:
why did this start failing in this PR? does this refer to a different repository?

Reply:
I remove that CounterTest.java. Should be replaced by new testkit.